resources/assets/components/utilities/NewsletterSubscription/SingleNewsletterSubscriptionForm.js

```diff
@@ -26,24 +30,44 @@ const SingleNewsletterSubscriptionForm = ({ emailSubscriptionTopic }) => {
   const handleOnFocus = event => {
     event.preventDefault();
 
-    // @TODO: add analytics tracking
-    console.log('handling input focus...');
+    trackAnalyticsEvent('focused_call_to_action_popover_email', {
```

Comment:
```suggestion
    trackAnalyticsEvent('focused_signup_newsletter', {
```

Looks like a few of the event names/labels below need to be updated as well 👾

Reply:
@mendelB ahhh shoot, i think the stuff in the pivotal ticket might be out of date and I just didn't double check it.

Reply:
Oh huh, yeah sorry I didn't even check the ticket since I assumed we'd change this one since it isn't at all a popover! Could this be a typo in the ticket? I'd assume we'd want these to be consistent with the `signup_newsletter` which seems to be how we're categorizing this component 🤔

Reply:
I asked to be sure in the ticket lol I don't think it was a typo but Diego D mentioned that he wasn't super set on the naming.

Reply:
I see now that there's discussion on that ticket -- I added in my vote. But otherwise, I suppose we keep this if that's the product wish 🤷

Reply:
Ah in that case, I think we have the freedom to just swap this 🎉